fmc/README.md

```diff
@@ -136,6 +136,7 @@ fields may not be changed or removed). Table revisions with different Major Vers
 | ldevid_tbs_size       | 2            | ROM        | Local Device ID TBS Size.                                                                                |
 | fmcalias_tbs_size     | 2            | ROM        | FMC Alias TBS Size.                                                                                      |
 | pcr_log_addr          | 4            | ROM        | PCR Log Address.                                                                                         |
+| pcr_log_index         | 4            | ROM        | Last empty PCR log entry slot index.                                                                     |
```

Comment:
also needs a more detailed description in the section just below this

Reply:
Done. I noticed there are quite a few fields missing, have added TODOs for them.

Reply:
Thanks, I've been trying to stay on top of this but I guess I didn't catch them all. Been meaning to go back and make sure the descriptions were all there.